packs/data/pathfinder-bestiary-2.db/army-ant-swarm.json

```diff
@@ -231,7 +231,7 @@
                 }
             },
             "img": "systems/pf2e/icons/actions/Passive.webp",
-            "name": "Scent",
+            "name": "Scent (imprecise) 30 feet",
```

Comment:
```suggestion
            "name": "Scent (Imprecise) 30 feet",
```

Reply:
It should have been in the merge 6 minutes before